Release/src/http/client/http_client_winrt.cpp

```diff
@@ -94,25 +94,33 @@ class HttpRequestCallback :
 
         utf16char *hdrStr = nullptr;
         HRESULT hr = xmlReq->GetAllResponseHeaders(&hdrStr);
-        if(hr != S_OK)
-        {
-            return hr;
-        }
-
-        auto progress = m_request->m_request._get_impl()->_progress_handler();
-        if (progress && m_request->m_uploaded == 0)
-        {
-            try { (*progress)(message_direction::upload, 0); } catch(...)
-            {
-                m_request->m_exceptionPtr = std::current_exception();
-                return ERROR_UNHANDLED_EXCEPTION;
-            }
-        }
-
-        web::http::details::parse_headers_string(hdrStr, response.headers());
-        m_request->complete_headers();
-
-        return S_OK;
+		if (SUCCEEDED(hr))
```

Comment:
The best thing is for you to submit this as a pull request to https://github.com/Microsoft/cpprestsdk and get them to take it first, then I'll just cherry pick that commit.